### ISSUE TITLE
I have problem when I write `Response` word in my markdown description.

### ISSUE THREAD

**Issue by AliSepehri** [a85/POSTMan-Chrome-Extension#1628]
I'm using markdown to write description for my APIs. If I have `Response` word as header in my description, a big white space appears after sending request! When I change `Response` word to something else problem disappears.

_For example:_

**Comment by pawanrawal** [comment]
Hi Ali

Could you please tell me what version of Postman are you using and send across a screenshot of this if possible ?

**Comment by AliSepehri** [comment]
@pawanrawal, today functionality has been changed and it doesn't show `Response` word.

##### Postman Info:

```
 Version 3.2.7 (packaged)
 OS: mac
 Architecture: x86-64
 Native client architecture: x86-64
```

#### My markdown description:

![markdown](https://cloud.githubusercontent.com/assets/7887290/12009135/ffc51430-ac80-11e5-844d-252b86e11cc8.png)

#### And result:

![result](https://cloud.githubusercontent.com/assets/7887290/12009137/29885c82-ac81-11e5-8f9a-bbfb0f47e47f.png)

**Comment by pawanrawal** [comment]
Ali

Thanks for reporting this.This is an issue with our markdown library. It auto-sets the content of the h4 element as its ID. We’ll fix this in version 3.2.9.
Let me know if you face any other issue.

**Comment by numaanashraf** [comment]
@AliSepehri We've addressed this issue in version 4.0+
